qgis_deployment_toolbelt/utils/journalizer.py

```diff
@@ -141,6 +143,10 @@ def headers():
         f"Certificate authority (CA) bundle to use: {getenv('REQUESTS_CA_BUNDLE', getenv('CURL_CA_BUNDLE'))}"
     )
 
+    if str2bool(getenv("QDT_SSL_USE_SYSTEM_STORES", False)):
+        truststore.inject_into_ssl()
```

Comment:
There is no request done in journalizer.py, is the call to inject_into_ssl() usefull here ?

Reply:
It's to be sure and avoid race conditions since `journalizer` is one of the first loaded modules where `requests` is imported.

But you're right, let's try without it.